src/scripts/breeding/Breeding.ts

```diff
@@ -223,7 +223,11 @@ class Breeding implements Feature {
     }
 
     public gainEggSlot() {
-        this.eggSlots += 1;
+        if (this.eggSlots < this.eggList.length){
+            this.eggSlots += 1;
+        } else {
+            console.error("eggSlots is larger than eggList.length.");
```

Comment:
Please refer to the docs to see how string templates work
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I've been using C# too much.